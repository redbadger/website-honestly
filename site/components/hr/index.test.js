import { shallow } from 'enzyme';
import React from 'react';
import HR from '.';

describe('HR', () => {
  const defaultProps = {
    color: 'red',
  };

  it('renders a red HR', () => {
    expect(shallow(<HR {...defaultProps} />)).toMatchSnapshot();
  });
});
