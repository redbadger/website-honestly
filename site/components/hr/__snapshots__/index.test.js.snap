// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HR renders a red HR 1`] = `
<hr
  className="hr red horizontal-line"
/>
`;
